### DISCUSSION TITLE
Calculate the azimuth and elevation of a star

### DISCUSSION THREAD

**Discussion by powersoft** [skyfielders/python-skyfield#745] [General]
I'm very new working with Skyfield. Until now I like to experience with converting my old Pyephem sources to Skyfield.

I have a probleem to calculate the azimuth and elevation of a particular star.
Have the Ra and Decl value of the that, the lat an Lon of my position on earth, but can't figure out to do the calculation.

Please can someone help me with an example.

Thank for any help.
Cheers, Jan Kromhout Hellevoetsluis-NL

**Comment by brandon-rhodes** [comment]
@powersoft — Take a look at this section of Skyfield's Examples page, it creates an ra/dec position and the position of an observer on Earth:

https://rhodesmill.org/skyfield/examples.html#when-is-a-body-or-fixed-coordinate-above-the-horizon

The only difference is that you'll want to do a normal `.observe()` call, rather than the more complicated `risings_and_settings()` call, and then ask for its azimuth and "altitude" (which is also called its "elevation" above the horizon):

https://rhodesmill.org/skyfield/positions.html#azimuth-and-altitude-from-a-geographic-position

Hopefully those two ideas combine correctly for you!
